packages/camera/android/src/main/java/io/flutter/plugins/camera/CameraPlugin.java

```diff
@@ -259,6 +259,7 @@ public boolean onRequestPermissionsResult(int id, String[] permissions, int[] gr
         final boolean enableAudio) {
 
       this.cameraName = cameraName;
+      this.enableAudio = enableAudio;
```

Comment:
I realized that this variable was never set in PR: https://github.com/flutter/plugins/pull/837

I hadn't realized a `null` `boolean` defaults to `false` in in java if statements.

Reply:
> I hadn't realized a `null` `boolean` defaults to false in in java if statements.

Yeah, there's a bunch of weird potential behavior around this. My favorite is that `Boolean` is a class and therefore nullable. But `boolean` is a primitive type and cannot be null. So it's totally possible to get an NPE casting a `Boolean` to a `boolean`.

```java
Boolean sneakyNull = someMethodThatCanReturnNull();
if (sneakyNull) { // evaluates to false
}
boolean value = sneakyNull; // null pointer exception
```

This is also true for `Integer` and `int`.